tests/importer/test_importer.py

```diff
@@ -68,6 +68,37 @@ def test_import_autocompiles():
     os.remove(get_bytecode_path(f.name))
 
 
+def test_import_failure_retryable():
+    "Test that a failed import does not prevents from re trying"
+    module_name = "again"
+
+    f = open("again.hy", "wb")
```

Comment:
@Kodiologist what would you think about starting to use the `pytester` plugin for this kind of stuff? Comes stock with pytest for this kind of stuff, just needs to be turned on:

```python
def test_import_failure_retryable(testdir):
    testdir.makefile("hy", again="""(import sys)""")
    testdir.syspathinsert()  # magically cleans up post test too

    import again
```

Keeps current working directory clean and makes sure everything is fresh each pytest run. I suspect stale .pyc files could cause a false pass here.

Reply:
I'm fine with using pytest plugins, sure; just make sure everything works.

Reply:
Okay, in which case `pytest_plugins = "pytester"` needs to go in our `conftest.py` and its loaded.